index.html

```diff
@@ -0,0 +1,72 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="style.css" />
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+    <title>Portfolio</title>
+  </head>
+  <body>
+    <div class="wrapper">
+      <header>
+        <nav class="top-bar">
+          <div class="logo">
+            <a href="#"><img src="My Logo.png" alt="logo"/></a>
+          </div>
+          <div class="menu-icon"> 
+            <a href=""><img src="./Union.svg" alt=""></a>          
+          </div>
+        </nav>
+        <div class="desktop-tools">
+          <ul class="navnav-menu">
+            <li class="navnav-item">
+              <a href="#Portfolio" class="link">Portfolio</a>
+            </li>
+            <li>
+              <a href="#About" class="link">About</a>
+            </li>
+            <li>
+              <a href="#Contact" class="link">Contact</a>
+            </li>
+          </ul>
+        </div>
+      </header>
+      <main>
+        <section class="headline">
+          <h1 class="title">I’m Samreen Glad to see you!</h1>
+          <p class="details">
+            I’m a software developer! I can help you build a product , feature
+            or website Look through some of my work and experience! If you like
+            what you see and have a project you need coded, don’t hestiate to
+            contact me.
+          </p>
+          <p></p>
+          <h2 class="contact-title">LET’S CONNECT</h2>
+          <ul class="social-media">
```

Comment:
- [OPTIONAL ] Good job of putting Your Social-media in a `ul` but  It could be better if you wrap it with a `div` tag for semantic reason :wink:  :smile: